cmd/get/clusters/command.go

```diff
@@ -0,0 +1,92 @@
+package clusters
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "clusters <cluster-id>"
+	alias = "cluster"
+
+	shortDescription = "Display one or many clusters"
+	longDescription  = `Display one or many clusters
+Output columns:
+- ID: Unique identifier of the cluster.
+- CREATED: Date and time of the Cluster CR creation.
+- CONDITION: Latest condition reported for the cluster. Can be "CREATING", "CREATED", "UPDATING", "UPDATED", "DELETING".
+- RELEASE: Release version of the cluster.
+- ORGANIZATION: Organization owning the cluster.
+- DESCRIPTION: User friendly description for the cluster.`
+
+	examples = `  # List all clusters you have access to
+  kgs get clusters
+  # Get one specific cluster by its ID
+  kgs get clusters f83ir
+  Note: 'kgs' is an alias for 'kubectl gs'.`
```

Comment:
Also suggesting some newlines for spacing / readability here

```suggestion
	examples = `  # List all clusters you have access to
  kgs get clusters
  
  # Get one specific cluster by its ID
  kgs get clusters f83ir
  
  Note: 'kgs' is an alias for 'kubectl gs'.`
```